### DISCUSSION TITLE
Use PYG packaged  model error RuntimeError

### DISCUSSION THREAD

**Discussion by xcgydfjjjderg** [pyg-team/pytorch_geometric#5885] [Q&A]

**Comment by rusty1s** [comment]
In which line does this error occur?

**Reply by xcgydfjjjderg** [reply]
The error code is x = F.elu(self.conv1(x, adjt))

**Reply by rusty1s** [reply]
Can you show me the full error log? That would be very helpful to track down.

**Reply by xcgydfjjjderg** [reply]
The code reports RuntimeError: Subtraction, the - operator, with a bool tensor is not supported. If you are trying to invert a mask, use the ~ or logical_not() operator instead.The error code is x = F.elu(self.conv1(x, adjt))，thank you.

**Reply by rusty1s** [reply]
Yeah, but is that the full log? There must be a traceback and a line number somewhere. I need this :)

**Reply by EdisonLeeeee** [reply]
I can indeed reproduce this bug. The code breaks here: https://github.com/pyg-team/pytorch_geometric/blob/da1f1876dbbfb5ecd1e01595353ce3f70267c42c/torch_geometric/nn/conv/supergat_conv.py#L191-L193 The reason is that `SuperGATConv` does not support `torch_sparse.SparseTensor` as input edge_index. Is there should be a fix for it?

**Reply by rusty1s** [reply]
Oh, I should have been more carefully in reading the code, as @EdisonLeeeee did. The fix seems to be indeed related to the usage of `SparseTensor`.

**Reply by rusty1s** [reply]
We will fix that in https://github.com/pyg-team/pytorch_geometric/pull/5888. For now, I suggest to not use `SparseTensor` for `SuperGATConv`.

**Reply by EdisonLeeeee** [reply]
You can simply install PyG from master and it should work for SparseTensor.
